### ISSUE TITLE
[1.12.2] Crash after loading "One of more entry values did not copy to the correct id"

### ISSUE THREAD

**Issue by aking4125** [TeamDman/PlusTiC#51]
Hello,

Upon adding this to our modpack. The game crashes with the following report:

[crash-2023-08-20_18.44.48-client.txt](https://github.com/TeamDman/PlusTiC/files/12389341/crash-2023-08-20_18.44.48-client.txt)

Log:

[debug.log](https://github.com/TeamDman/PlusTiC/files/12389340/debug.log)

Any ideas?

**Comment by TeamDman** [comment]
the only thing plustic related in the stack trace is that it's in the mod list at all, which doesn't mean it's not the mod's fault it just means the problem is a little more esoteric then normal.

`java.lang.RuntimeException: One of more entry values did not copy to the correct id. Check log for details!` could mean that plustic is adding an entry to a registry that another mod doesn't like, but I don't really have any ideas for what is happening beyond that.

There's a lot of ASM and mixins going on? Probably an incompatibility with another mod

**Comment by aking4125** [comment]
Yeah this is a strange one. Removing it causes the pack to load normally. I'll try and start the removal process a few mods at a time to see which conflict occurs.

> There's a lot of ASM and mixins going on? Probably an incompatibility with another mod

Yeah lol. I guess newer stuff for 1.12 is using that quite a lot. I tried removing a few like Universal tweaks and MUP but nada so far. I'll be honest it may take me some time to see where the conflict lies but I'll do my best with my limited schedule.

Thank you for the response!
